package.json

```diff
@@ -2,7 +2,7 @@
   "name": "meetup-web-platform",
   "version": "0.0.0",
```

Comment:
Does travis actually update the package.json version? i.e., will i see it in `node_modules/meetup-web-platform/package.json`?